.circleci/config.yml

```diff
@@ -17,6 +17,22 @@ references:
         pacman -Syu --noconfirm python python-pip git base-devel curl
         pip install coveralls coverage
 
+  ubuntu_deps_opencv: &ubuntu_deps_opencv
+    run:
+      name: Install opencv dependencies on Ubuntu.
+      command: |
+        export DEBIAN_FRONTEND=noninteractive
+        apt-get install -y --force-yes opencv-data libopencv-dev python3-opencv
+        ldconfig
+        echo 'export OPENCV=true' >> $BASH_ENV
+
+  arch_deps_opencv: &arch_deps_opencv
+    run:
+      name: Install opencv dependencies on Arch Linux.
+      command: |
+        pacman -Syu --noconfirm opencv opencv-samples hdf5 gtk3
```

Comment:
Do we need `opencv-samples` ?

Reply:
I might try without it and we'll se, this configuration was what I found on stackoverflow discussion